### DISCUSSION TITLE
Lost NFC

### DISCUSSION THREAD

**Discussion by afarkassk** [driedpampas/realme-8-megaguide#27] [Q&A] [answered]
Hi, i followed your guide to unlock Realme 8 bootloader and i succeeded. However, the used rom to upgrade to RUI3 is indian version, which has no NFC... Now i have no NFC on my phone and im unable to find any EU version rom with NFC support that could be flashed via sp_flas_tool... Maybe you could point me to one if you know about?
Thank you

**Comment by HowWof** [comment, marked as answer]
https://drive.google.com/file/d/1dFKOSI9QllJgI5uL2jW-C1CdFG_rvn-y/view?usp=share_link
C.11 (Realme UI 3 Global)

**Reply by afarkassk** [reply]
Oh, miracles happens. Thank you so much. May the lord bless you and all of your beloved ones

**Comment by driedpampas** [comment]
@afarkassk we also have a telegram group [t.me/Realme8AOSPGroup](https://t.me/Realme8AOSPGroup)
(and yes I pinned it in case anyone else has the same problem)

**Reply by afarkassk** [reply]
Great. I gave a try to telegram and iv got limited immediatelly after the first message. I was asking whether there is an alternative lk patcher site, as the one in the tutorial does not work and the manual patcher returns error, but my message never got to the chat, i wasw limited immediatelly.

**Reply by HowWof** [reply]
Write to me @HowWof on telegram
